annubes/task.py

```diff
@@ -243,30 +230,26 @@ def _build_trials_inputs(self) -> NDArray[np.float32]:
             (self._ntrials, len(self.time), self.n_inputs),
             dtype=np.float32,
         )
-        sel_value_in = np.full(  # TODO: needs a better name
-            (self._ntrials, self.n_inputs - 1),  # should not include start cue
-            0,
-            dtype=np.float32,
-        )
 
         for n in range(self._ntrials):
             for idx, m in enumerate(self.modalities):
                 if (self._modality_seq[n] != "catch") and (m in self._modality_seq[n]):
-                    sel_value_in[n, idx] = self._rng.choice(self.stim_intensities[1:], 1)
-                sel_value_in[n, idx] = self._rescale(sel_value_in[n, idx], self.rescaling_coeff)
-                x[n, self._phases["input"], idx] = sel_value_in[n, idx]
-                x[n, self._phases["fix_time"], idx] = self._rescale(
-                    self.fix_intensity,
-                    self.rescaling_coeff,
-                )
+                    value = self._rng.choice(self.stim_intensities, 1)
+                else:
+                    value = 0
+                x[n, self._phases["input"], idx] = value
+                x[n, self._phases["fix_time"], idx] = self.fix_intensity
             x[n, self._phases["input"], self.n_inputs - 1] = 1  # start cue
 
         # generate noise
         alpha = self.dt / self.tau
         noise_factor = self.noise_std * np.sqrt(2 * alpha) / alpha
-        noise = noise_factor * self._rng.normal(loc=0, scale=1, size=x.shape)
+        x += noise_factor * self._rng.normal(loc=0, scale=1, size=x.shape)
```

Comment:
Should we add a check somewhere that neither `dt` nor `tau` are 0 and return a better warning than the one that would be returned here if that would happen?

Also, maybe we can change the comment on line 244 to `add noise`.